### PR TITLE
- BugFix: Logo image size

### PR DESCRIPTION
**Pull request by MykolaYurochko**
image

### CODE REVIEW

File: src/app/component/layout/components/header/header.component.scss [hunk 1]
```diff
@@ -472,6 +472,12 @@ $screen-320: " screen and (max-width : 320px)";
 }
 
 @media #{$screen-870} {
+  .header{
```
Comment (HryshkoAnton) [inline src/app/component/layout/components/header/header.component.scss:475]: Expected single space before "{"

File: src/app/component/layout/components/header/header.component.scss [hunk 2]
```diff
@@ -472,6 +472,12 @@ $screen-320: " screen and (max-width : 320px)";
...
+    .logo{
```
Comment (HryshkoAnton) [inline src/app/component/layout/components/header/header.component.scss:476]: Expected single space before "{"

File: src/app/component/layout/components/header/header.component.scss [hunk 3]
```diff
@@ -504,11 +510,7 @@ $screen-320: " screen and (max-width : 320px)";
     }
   }
 
-  .header {
-    .logo {
-      margin-top: 15px;
-    }
-  }
+
```
Comment (HryshkoAnton) [inline src/app/component/layout/components/header/header.component.scss:513]: Unexpected whitespace at end of line

Comment (MykolaYurochko) [reply src/app/component/layout/components/header/header.component.scss]: Fixed!

Comment (MykolaYurochko) [reply src/app/component/layout/components/header/header.component.scss]: Fixed!

File: src/app/component/layout/components/header/header.component.scss (hunk 2 above)
Comment (MykolaYurochko) [reply src/app/component/layout/components/header/header.component.scss]: Fixed!

File: src/app/component/layout/components/header/header.component.scss (hunk 1 above)
Comment (MykolaYurochko) [reply src/app/component/layout/components/header/header.component.scss]: Fixed!

**Review by yulkats — APPROVED** [review]

**Pull request merged**